feat(M&B): FAQ for event retention period (#5207)
Kong/developer.konghq.com: app/_includes/faqs/metering-event-retention.md

{% if include.section == "question" %}
How long are metering events retained in {{site.konnect_short_name}}?
{% elsif include.section == "answer" %}

> By default, metering events are retained for 2 years. This is not configurable.

Author: ```suggestion
By default, metering events are retained for 2 years. You cannot configure this.
```

> By default, metering events are retained for 2 years. You cannot configure this.

Reviewer: ```suggestion
By default, metering events are retained for two years. You cannot configure this.
```
